kubectl.tf

```diff
@@ -1,3 +1,24 @@
+resource "null_resource" "install_kubectl" {
+  provisioner "local-exec" {
+    working_dir = "${path.module}"
+
+    command = <<EOH
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v${local.kubectl_versions[var.cluster_version]}/bin/linux/amd64/kubectl && \
```

Comment:
I don't know what this is but it's nothing related to the title of this PR or the description. Please fix this to keep the description and the changes concise 🙂

Reply:
the commits unrelated to this PR are removed. Please review.